numba/cuda/cudadrv/driver.py

```diff
@@ -185,8 +200,13 @@ def __init__(self):
             self.initialization_error = e
 
     def initialize(self):
+        # lazily initialize logger
+        global _logger
+        _logger = _make_logger()
```

Comment:
Hmm, you still should do the initialization only once (because `addHandler` adds a handler every time it is called...).